### PR TITLE
many: fixes related to inclusive language

### PR DESCRIPTION
**Pull request by stolowski**
Fixes related to inclusive language.

One notable change is the rename of "dummy" interface to "empty" (this shouldn't be used by real snaps).

### CODE REVIEW

**Review by mardy — APPROVED** [review]
LGTM! There are a couple of other naming failures, though :-)

**Review by bboozzoo — APPROVED** [review]
LGTM

**Review by MiguelPires — APPROVED** [review]
LGTM

File: secboot/secboot_tpm.go [hunk 1]
```diff
@@ -415,7 +415,7 @@ func buildPCRProtectionProfile(modelParams []*SealKeyModelParams) (*sb_tpm2.PCRP
 			PCRAlgorithm:  tpm2.HashAlgorithmSHA256,
 			LoadSequences: loadSequences,
 			// TODO:UC20: set SignatureDbUpdateKeystore to support applying forbidden
-			//            signature updates to blacklist signing keys (after rotating them).
+			//            signature updates to excluded signing keys (after rotating them).
```
Comment (mardy) [inline secboot/secboot_tpm.go:418]: There's a little change of meaning here. To maintain the old meaning, we should use "exclude" here. Or was the old text wrong?

**Review by mardy — APPROVED** [review]
LGTM, with a question.

File: secboot/secboot_tpm.go (hunk 1 above)
Comment (stolowski) [reply secboot/secboot_tpm.go]: You are right, I think 'exclude' makes more sense?

**Pull request merged**